src/user-guide/running-workflows/disaster-recovery.rst

```diff
@@ -0,0 +1,64 @@
+.. TODO - Re-write for Cylc 8
+
+
+.. _Disaster Recovery:
+
+Disaster Recovery
+-----------------
+
+If a run directory gets deleted or corrupted, the options for recovery
+are:
+
+- restore the run directory from back-up, and restart the workflow
+- re-install from source, and warm start from the beginning of the
+  current cycle point
+
+A warm start (see :ref:`Warm Start`) does not need the workflow database, but it
+could re-run a significant number of tasks that had already completed.
+
+To restart the workflow, the critical Cylc files that must be restored are:
+
+.. code-block:: sub
+
+   # On the workflow host:
+   ~/cylc-run/WORKFLOW-NAME/
+       flow.cylc  # installed workflow configuration
+       log/db  # public workflow DB (can just be a copy of the private DB)
+       log/rose-suite-run.conf  # (needed to restart a Rose workflow)
+       .service/db  # private workflow DB
+       .service/source -> PATH-TO-WORKFLOW-DIR  # symlink to workflow source
```

Comment:
Wondering if it would be worth mentioning the auth keys here and on the job host? They are necessary for communication between the two so if a user deletes them by mistake, they will need to be recreated by cylc 
 running a process that runs remote init, ie. cylc play or cylc restart.

Reply:
This section needs to be re-written #268

Reply:
Deliberately not changed content: Diff might be tricky to read.